nixos/modules/services/networking/netbird/signal.nix

```diff
@@ -59,19 +75,21 @@ in
       wantedBy = [ "multi-user.target" ];
 
       serviceConfig = {
-        ExecStart = escapeSystemdExecArgs [
-          (getExe' cfg.package "netbird-signal")
-          "run"
-          # Port to listen on
-          "--port"
-          cfg.port
-          # Log to stdout
-          "--log-file"
-          "console"
-          # Log level
-          "--log-level"
-          cfg.logLevel
-        ];
+        ExecStart =
+          escapeSystemdExecArgs [
+            (getExe' cfg.package "netbird-signal")
+            "run"
+            # Port to listen on
+            "--port"
+            cfg.port
+            # Log to stdout
+            "--log-file"
+            "console"
+            # Log level
```

Comment:
```suggestion
            "--port"
            cfg.port
            "--log-file"
            "console"
```

Reply:
Any particular reason to remove the comments? I didn't touch them since the intent was for this to be a small, simple fix focused on the port clash, but I'm open to removing them if needed.